examples/python/interface.py

```diff
@@ -0,0 +1,158 @@
+        relative path to custom KPI (e.g., 'custom_kpi.custom_kpis_example.config')
+        default is None.
+    forecast_config: list, str
+        List of strings.  Each element is a point that will be passed to the /forecast restful call.
+        Default is None.
+
+    Returns
+    -------
+    kpi : dict
+        Dictionary of core KPI names and values.
+        {kpi_name : value}
+    res : dict
+        Dictionary of trajectories of inputs and outputs.
+    custom_kpi_result: dict
+        Dictionary of tracked custom KPI calculations.
+        Empty if no customized KPI calculations defined.
+
+    """
+    # SETUP TEST CASE
+    # ---------------
+    # Set URL for testcase
+    url = 'http://127.0.0.1:5000'
+    # Set simulation parameters
+    control = Control(control_module, forecast_config, step)
+    # GET TEST INFORMATION
+    # --------------------
+    print('\nTEST CASE INFORMATION\n---------------------')
+    # Test case name
+    name = requests.get('{0}/name'.format(url)).json()
+    print('Name:\t\t\t\t{0}'.format(name))
+    # Inputs available
+    inputs = requests.get('{0}/inputs'.format(url)).json()
+    print('Control Inputs:\t\t\t{0}'.format(inputs))
+    # Measurements available
+    measurements = requests.get('{0}/measurements'.format(url)).json()
+    print('Measurements:\t\t\t{0}'.format(measurements))
+    # Default simulation step
+    step_def = requests.get('{0}/step'.format(url)).json()
+    print('Default Simulation Step:\t{0}'.format(step_def))
+
+    # Define customized KPI if any
+    custom_kpis = []  # Initialize customized kpi calculation list
+    custom_kpi_result = {}  # Initialize tracking of customized kpi calculation results
+    if customized_kpi_config is not None:
+        with open(customized_kpi_config) as f:
+            config = json.load(f, object_pairs_hook=collections.OrderedDict)
+        for key in config.keys():
+            custom_kpis.append(CustomKPI(config[key]))
+            custom_kpi_result[CustomKPI(config[key]).name] = []
+    custom_kpi_result['time'] = []
+    # --------------------
+
+    # RUN TEST CASE
+    # -------------
+    start = time.time()
+    # Initialize test case
+    print('Initializing test case simulation.')
+    res = requests.put('{0}/initialize'.format(url), data={'start_time': 0, 'warmup_period': 0}).json()
+    if res:
+        print('Successfully initialized the simulation')
+    print('\nRunning test case...')
+    # Set simulation step
+    res = requests.put('{0}/step'.format(url), data={'step': step})
+    # Initialize u
+    u = control.initialize()
+    # Store prediction if any
+    # Simulation Loop
+    forecast = None
+    for i in range(int(length/step)):
+        # Advance simulation
+        y = requests.post('{0}/advance'.format(url), data=u).json()
+        if control.use_forecast:
+            forecast_data = requests.get('{0}/forecast'.format(url)).json()
+            # Compute next control signal

```

Comment:
Incorrect comment for next line of code.

Reply:
I updated the comment

Reply:
Thank you!